### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Setup_Linux_Usability:The "Installation" header has an additional "# installation link" besides it.

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/vscode-docs#5549]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yUsable;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win10;#FTP;#DesktopWeb;#A11ySev4;#Chrome;#Keyboard;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win10
Chrome Version: Version 103.0.5060.134(Official build)(64-bit)

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Tab till "Docs" link.
4. From left navigation, select "Setup".
5. Tab till"Linux" link.
6. Tab till link given beside heading "Installation".

## Actual:
The "Installation" header has an additional "# installation link" besides it 
**Similar issue is observed throughout the application.**

## Expected:
The link "# Installation" can directly be associated to the heading "Installation" so that keyboard user will not require to press one more Tab key to visit the "#" link. They can directly Tab to the heading "Installation" and click on the link.

## User Impact:
Screen Reader users will hear the word installation twice.

## Attachment
![A11y_Docs Setup_Linux_Usability](https://user-images.githubusercontent.com/93374043/183062603-991196af-7d48-4014-87d5-9591ff88be42.gif)

**Comment by rzhao271** [comment]
@Yashi1201 are you saying the entire `Installation #` string should be a link?
A concern I have is that if the entire header is a link, then when focused, the entire header will be underlined. Some headers such as https://code.visualstudio.com/docs/setup/linux#_visual-studio-code-is-unable-to-watch-for-file-changes-in-this-large-workspace-error-enospc are over a line long.

Also, when I tab through the page on Microsoft Edge, the tab focuses only on the `#` parts each time.

**Comment by rzhao271** [comment]
Office hour conclusions
- Yes we want to wrap the entire header in a link so that there's no risk of the screenreader reading the header and the anchor link's aria text afterwards (aka reading the header twice)
- We can style the link to be the same as it is now, meaning it doesn't need a massive underline and it can stay black
